drivers/adult_only_households_sub_pop/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -0,0 +1,23 @@
+module AdultOnlyHouseholdsSubPop::GrdaWarehouse::Hud
+  module ClientExtension
+    extend ActiveSupport::Concern
+    include ArelHelper
+
+    included do
+      scope :adult_only_households, -> do
+        where(
```

Comment:
What are the trade-offs for this vs join/merge?

Reply:
So, this would be faster than an `id in ()` and only returns one record per client unlike a join or merge.  No distinct necessary!  I think there's a chance it's faster than a join too because postgres may be able to optimize the exists query independently.